### DISCUSSION TITLE
Allow multiple proxied SMTP ports

### DISCUSSION THREAD

**Discussion by Wouter0100** [postalserver/postal#1644] [Feature suggestions]
I'm currently setting up Postal, and I have a loadbalancer in front of Postal's SMTP servers. This loadbalancer forwards both port 25 and port 2525 to Postal's SMTP server, that is configured to run on 5025.

It would be awesome to allow the Postal SMTP server to continue to run on 5025, but have the option to override the SMTP server ports for visual purpose. Currently the SMTP port is "hardcoded" to be the actual SMTP server is running on [here](https://github.com/postalserver/postal/blob/4404b3e02c1722808157c3f590310ead9e28641d/app/views/help/outgoing.html.haml#L33-L36).

What would you guys think of this feature request? If okay, I'll open a PR for it.

**Comment by willpower232** [comment]
I think that for the purposes of documenting to the end-user what ports and details they can expect Postal to be available on, it makes sense for that page to be overridable in case of setups like yours. As long as your change defaults to the current behaviour then it sounds great.
